docs/manual/java/gettingstarted/index.toc

```diff
@@ -1,5 +1,5 @@
 Installation:Installing Lagom
-GettingStarted:Getting started with Lagom
-GettingStartedMaven:Getting started with Lagom in Maven
+GettingStarted:Getting started with Lagom;next=GettingStartedMaven,GettingStartedSbt
+GettingStartedMaven:Getting started with Lagom in Maven;next=IDEs
 GettingStartedSbt:Getting started with Lagom in Sbt
```

Comment:
@rstento This is what it will look like to have multiple and/or custom next links, so the first getting started page links to both the maven and sbt pages, while the maven page skips the sbt page as a next link and goes straight to IDEs.